homeassistant/components/philips_air_purifier/client.py

```diff
@@ -0,0 +1,337 @@
+"""client provides a ReliableClient for Philips Air Purifiers."""
+from __future__ import annotations
+
+import asyncio
+from asyncio.tasks import FIRST_COMPLETED
+from collections.abc import Callable
+from datetime import datetime, timedelta, timezone
+import enum
+from enum import Enum
```

Comment:
```suggestion
from enum import Enum, unique
```

My preference would be this probably. Or just import enum instead of importing it in two different ways :).